improver/cli/snow_fraction.py

```diff
@@ -58,4 +58,6 @@ def process(*cubes: cli.inputcube, model_id_attr: str = None):
     from improver.precipitation_type.snow_fraction import SnowFraction
```

Comment:
Apologies @withshubh, but we now have snow fraction twice? Everything else looks good.